From: Bart Van Assche <bart.vanassche@sandisk.com>
Date: Tue, 14 Jun 2016 17:03:45 +0200
Subject: block/bio-integrity.c: Add #include "blk.h"
References: bnc#1003941,FATE#321732
Patch-Mainline: v4.8-rc1
Git-commit: 1179a5a0851be6753b43601c00602dcef99e68f7

This patch avoids that building with W=1 C=2 triggers the following
warning:

block/bio-integrity.c:35:6: warning: symbol 'blk_flush_integrity' was not declared. Should it be static?

Signed-off-by: Bart Van Assche <bart.vanassche@sandisk.com>
Signed-off-by: Jens Axboe <axboe@fb.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 block/bio-integrity.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/block/bio-integrity.c b/block/bio-integrity.c
index 711e4d8d..15d37b1 100644
--- a/block/bio-integrity.c
+++ b/block/bio-integrity.c
@@ -26,6 +26,7 @@
 #include <linux/bio.h>
 #include <linux/workqueue.h>
 #include <linux/slab.h>
+#include "blk.h"
 
 #define BIP_INLINE_VECS	4
 
-- 
1.8.5.6

